autowebcompat/network.py

```diff
@@ -116,7 +116,7 @@ def create_inception_network(input_shape):
     return Model(input, x)
 
 
-def create(input_shape, network='vgglike', weights=None):
+def create(input_shape, network, weights=None):
     network_func = globals()['create_%s_network' % network]
```

Comment:
Add an assertion at the beginning that the network is in the list of supported networks. Let's leave the default as it was.

Reply:
sir, this line in train.py or pretrain.py
```parser.add_argument("network", type = str, choices = SUPPORTED_NETWORKS, help = "Select the supported network")```
 already "choices"  checks if the given input 'network' is in the list or not 
if it is not present then it throws error 
``` pretrain.py: error: argument network: invalid choice: 'maxnet' (choose from 'inception', 'vgglike', 'vgg16')``` where maxnet is the given argument.
do we still need to add assertion to it ?

Reply:
Yes, because this module could be in the future used by other scripts too, so it should not depend on what the callers do.

Reply:
Added assertion in network.py